base/src/main/proto/spine/options.proto

```diff
@@ -296,7 +296,43 @@ extend google.protobuf.MessageOptions {
     // The option to mark a message as state of an entity.
     EntityOption entity = 73903;
 
-    // reserved 73904 to 73910 for future validation options
+    // A validation rule for a field declaration.
+    //
+    // Allows to define validation options for fields of the field declaration.
+    // If a specified options is already defined in a target message,
+    // then options from the validation rule will be used.
+    //
+    // Validation rules are applicable only for field declarations of a message type.
+    //
+    // Validation rule target should be marked as `valid`. Otherwise a rule will be ignored.
+    //
+    // Target of a validation rule should be specified with the following pattern:
+    // `package.MessageName.field_name`.
+    //
+    // For example, you have a message that should be used in several places,
+    // that are impose context-dependent validation rules. The definitions will look like:
+    //
+    // message StringChange {
```

Comment:
Please have a package in the example as well — so that you could reference it below instead of `package. ...`. Something like `io.spine.example` will do.